docs/release-process.rst

```diff
@@ -0,0 +1,19 @@
+===============
+Release Process
+===============
+
+Actual mechanics of making the release:
+
+- Update the changelog manually, and commit the changes.
+- Install :pypi:`bump2version`::
+
+    pip install bump2version
+
+- Run it to bump the version strings within the project, like::
+
+    bump2version minor
+
+- Push the commit and tags to GitHub.
+- To build and upload the distribution files, run::
+
+    flit publish
```

Comment:
```suggestion
  A GitHub action will upload the release to PyPI.
```

(Though I'll need to do one manually after the rename and then update the token on Github)